src/ips/algorithm.rs

```diff
@@ -465,8 +466,8 @@ impl Ips {
         let mut graph = construct_graph(nodes);
 
         if generate_full {
-            let betweenness = graph.betweenness_centrality();
-            let closeness = graph.closeness_centrality();
+            let betweenness = graph.betweenness_centrality(NUM_THREADS, false);
```

Comment:
general API comment:
it's impossible to say what `false` here means without checking the function documentation - so it would be good to replace it with an enum value

...5 minutes later, after I did my documentation check, here's a suggestion which would make it clear on the spot:
```rust
let betweenness = graph.betweenness_centrality(NUM_THREADS, Normalization::Enable);    // or NormalizationOpt::On
```

Reply:
1. Hmm, well I see clearly (at least in VS Code) that the the parameter is called `normalize`
![Screenshot 2023-03-30 at 10 10 20](https://user-images.githubusercontent.com/15172321/228773422-f3e82e77-91d8-4011-a5eb-2439c3b66cbe.png)


2. I move the mouse over the function name and get this:
![Screenshot 2023-03-30 at 10 13 32](https://user-images.githubusercontent.com/15172321/228773010-2faf3b19-b979-442d-b032-749dd4609c86.png)
08.png)

3.  > it's impossible to say what false here means without checking the function documentation

Well, I would also say it's also impossible to use the function in your enum-instead-of-bool-case without checking the documentation.  Except in your case, to need to know an additional parameter type.

4.  Other public functions in the spectre API also use bools.

![Screenshot 2023-03-30 at 10 25 47](https://user-images.githubusercontent.com/15172321/228776198-577f18bd-6df5-4bbd-9408-44ce0ac8a242.png)



5.  IMO, I don't see the point of using an enum type for `bool`, instead of just plain `bool`.  More text, more stuff to know, more complicated.  
6.  In my experience, the meaning of a boolean parameter is usually quite clear from the name. 
😄

Reply:
I can fix this with next PR.  Since it's really a kind of constant -- just like any other **magic** number or string -- I'd define this in `constants.rs` (which was just added for the `NUM_THREADS` define).  I prefer a const name to an enum.
```
const NORMALIZATION_OFF: bool = false;
```

Reply:
> Hmm, well I see clearly (at least in VS Code) that the the parameter is called normalize

Okay, but that's the developer's perspective with VS Code - not everybody uses that IDE. The clean API approach shouldn't consider the IDE tools.

but the more important perspective:
I couldn't see anything like that while doing the code review on GitHub. I had to manually find the repo and seek the parameter there 🤷‍♂️

Reply:
>I can fix this with next PR. Since it's really a kind of constant -- just like any other magic number or string -- I'd define this in constants.rs (which was just added for the NUM_THREADS define). I prefer a const name to an enum.
>
>> const NORMALIZATION_OFF: bool = false;

it's another approach, but then the user of the API needs to ensure clean usage of the API - still not the cleanest solution, but it would definitely give much more context